crates/sp-consensus-subspace/src/lib.rs

```diff
@@ -228,5 +230,10 @@ sp_api::decl_runtime_apis! {
 
         /// Returns `Vec<RootBlock>` if a given extrinsic has them.
         fn extract_root_blocks(ext: &Block::Extrinsic) -> Option<Vec<RootBlock>>;
+
+        /// Extract block object mapping for a given block
+        /// TODO: remove once the node is upgraded
+        #[deprecated = "Use ObjectsApi function with same name"]
+        fn extract_block_object_mapping(block: Block, validated_object_calls: Vec<Hash>) -> BlockObjectMapping;
```

Comment:
Can't do that. We need to revert this API to this state: https://github.com/subspace/subspace/pull/418/files#diff-ed6f1fa9156c1cb2a09d7368f6e1d6e4f2b32d6e72b2aa57ee0a13c4d580b53cL234

And make client fall back from one to another somehow. Or else client will fail to sync from existing network.

Reply:
oops! yes

Reply:
Hmm, why doesn't work? Did I miss something? @vedhavyas